dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -75,6 +76,45 @@
    */
   public static void main(String[] arguments)
       throws IOException, ClassNotFoundException, RepositoryException {
+    List<Path> jarFilePaths = parseArguments(arguments);
+    if (jarFilePaths == null) {
+      System.err.println("Could not parse arguments to jar files.");
```

Comment:
This seems redundant with what `parseArguments` already prints out.

Reply:
removed.